Sources/BraintreeCore/RepeatingTimer.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+final class RepeatingTimer {
```

Comment:
I like this idea of moving the timer wrapping & management logic into it's own class